Add doc improvements related to environment and IdP configurations for Dev Portal (#1793)
wso2/docs-choreo-dev: en/developer-docs/docs/administer/configure-developer-portal-idps.md

> # Configure Identity Providers for Developer Portal

Reviewer: same here

You can manage which Identity Providers (IdPs) are available for consumers to authenticate and generate API keys in the Developer Portal.

!!! Note -

>     For organizations created after <date>, the Choreo Built-in Identity Provider is configured by default. Organizations created before this date allow all configured Identity Providers to be used in the Developer Portal. 

Reviewer 2: ```suggestion
    For organizations created after <date>, the Choreo Built-in Identity Provider is configured by default. Organizations created before this date allow all configured identity providers to be used in the Developer Portal. 
```

Follow these steps to configure Identity Providers:

> 1. Navigate to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/) and sign in using your Google, GitHub, or Microsoft account.

Reviewer 2: ```suggestion
1. Sign in to [Choreo](https://console.choreo.dev/).
2. In the Choreo Console, go to the top navigation menu and click **Organization**. Then select your organization.
```

2. Select your organization from the **Organization** dropdown in the Choreo Console header.

> 3. From the left navigation menu, select **API Management** > **Developer Portal**.

Reviewer 2: ```suggestion
3. In the left navigation menu, click **API Management** and then click **Developer Portal**.
```

4. In the **Identity Providers** section, specify which identity providers should be used for both the **Sandbox** and **Production** environments of the Developer Portal.

> !!! Note -

Reviewer: would it be better if we point to that page from here. wdyt?

Author: We don't have a document related this specific configuration.

> 4. In the **Identity Providers** section, specify which Identity Providers are allowed for both the **Sandbox** and **Production** environments of the Developer Portal.

Reviewer 2: ```suggestion
4. In the **Identity Providers** section, specify which identity providers should be used for both the **Sandbox** and **Production** environments of the Developer Portal.
```

>     Any Identity Provider configured for the Developer Portal's **Production** environment must also be enabled in the Choreo **Production** environment.

Reviewer 2: ```suggestion
    Any identity provider configured for the Developer Portal's **Production** environment must also be enabled in the Choreo **Production** environment.
```

# Configure Identity Providers for Developer Portal

You can manage which Identity Providers (IdPs) are available for consumers to authenticate and generate API keys in the Developer Portal.

!!! Note -
    For organizations created after <date>, the Choreo Built-in Identity Provider is configured by default. Organizations created before this date allow all configured identity providers to be used in the Developer Portal. 

Follow these steps to configure Identity Providers:

1. Sign in to [Choreo](https://console.choreo.dev/).

> 2. Select your organization from the **Organization** dropdown in the Choreo Console header. 

Reviewer 2: ```suggestion
3 In the Choreo Console, go to the top navigation menu and click **Organization**. Then select your organization. 
```

> 3. Click on the selected organization on top navigation bar. This takes you to the Organization's overview page.

Reviewer 2: ```suggestion
```
